apps/elewa-website/src/styles.scss

```diff
@@ -1,42 +0,0 @@
-/* You can add global styles to this file, and also import other style files */
-
-@import '_variables';
-
-* {
-  box-sizing: border-box;
-  margin: 0;
-  padding: 0;
```

Comment:
This removes the default styles from the site.

Reply:
changed file to as of main repository